worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -234,6 +234,16 @@ private Extent wrapExtent(Extent extent, EventBus eventBus, EditSessionEvent eve
         return event.getExtent();
     }
 
+    // pkg private for TracedEditSession only
+
+    ChunkBatchingExtent getChunkBatchingExtent() {
```

Comment:
Would it be better to just have a `commitRequired()` method here? That way we could have something semi-generic that also allows for custom extents.

Reply:
Sure, I'll do that.

Reply:
Sure, I'll do that.